### PR TITLE
Import all available header properties from Postman

### PR DESCRIPTION
**Pull request by petoc**
changelog(Improvements): Added support to import "disabled" and "description" header properties from Postman.

Importing header properties "disabled" and  "description" in Postman importer.

### CODE REVIEW

**Review by jackkav — APPROVED** [review]
Nice job 👍 Thanks

**Comment by petoc** [comment]
Sorry, I did not run the whole test suite, so missed fixtures. Tests should pass now.

File: packages/insomnia/src/utils/importers/importers/postman.ts [hunk 1]
```diff
@@ -136,9 +136,11 @@ export class ImportPostman {
       url: this.importUrl(request.url),
       parameters: parameters,
       method: request.method || 'GET',
-      headers: headers.map(({ key, value }) => ({
+      headers: headers.map(({ key, value, disabled, description }) => ({
         name: key,
         value,
+        disabled: disabled || false,
+        description,
```
Comment (jackkav) [inline packages/insomnia/src/utils/importers/importers/postman.ts]:
```suggestion
        ...(typeof disabled !== 'undefined' ? { disabled } : {}),
        ...(typeof description !== 'undefined' ? { description } : {}),
```
In the insomnia RequestHeader type these fields are optional so make them conditionally inserted would best fit that type. Then you need not change the tests that don't test your change, and you can revert that comit.

**Comment by petoc** [comment]
Thanks for comment. I reverted changes to other tests and incorporated your suggestions.

**Comment by jackkav** [comment]
disabled will omit the field if its true, check my diff suggestion

**Comment by petoc** [comment]
Initially I checked disabled against undefined, as you suggested, but wsdl importer for example is setting disabled=false for its headers, while tests do not test it. So they would fail. Also I think, if disabled is false (and is optional) it could be omited as it is default behavior.

**Comment by jackkav** [comment]
The issue appears to have been bad test fixtures containing `disabled:false` that was previously ignored.

**Comment by filfreire** [comment]
@petoc you can claim a free tshirt for this PR contribution - find more at https://konghq.com/community/open-source-contribution

**Pull request merged**